src/app/recents/recents.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { TabsModule } from 'ngx-bootstrap/tabs';
+
+import { NodeTypeDisplayPipe } from '../pipes/nodedisplay';
+import { RecentsService } from '../services/recents.service';
+
+
+@Component({
+  selector: 'app-recents',
+  templateUrl: './recents.component.html',
+  styleUrls: ['./recents.component.css']
+})
+export class RecentsComponent implements OnInit {
+
+  recentsAll: Array<any>; // all recents of every type
+  meta: Array<any>; // metadata
```

Comment:
Could you please expand the comment of ` // metadata` or remove it? I feel like the variable name describes it as well as the comment.